en/core-api/direct-debit.md

```diff
@@ -518,10 +518,17 @@ The sample HTTP notification request received at merchant backend for *Direct De
 #### [HTTP(S) Notification/Webhooks](/en/after-payment/http-notification.md)
 </div>
 
-## Switching to Production Environment
-Follow the steps given below to switch to Midtrans *Production* environment and to accept real payments from real customers.
-1. Change API domain URL from `api.sandbox.midtrans.com` to `api.midtrans.com`.
-2. Use *Client Key* and *Server Key* for *Production* environment. For more details, refer to [Retrieving API Access Keys](/en/midtrans-account/overview.md#retrieving-api-access-keys).
+### 5. Switching to Production Mode
```

Comment:
Why change heading structure from `##` to `### 5.` ? IMO it shouldn't be part of integration step's sub heading, as it would seems that the integration steps is now longer.

Reply:
Thanks for the feedback Da. I get the details from Yana, about the merchant that facing 401. They claim they weren't found the information in the integration steps and that they should change the environment to production before going live. That's why I put the section going live, into integration steps. But I will let you decide what the best to choose ya. 

Thank you

Reply:
hmm I think make sense, please proceed with your approach for now 👌  we can re-evaluate sometimes later if we need to.